File: dmn-test-cases/signavio/dmn/dmn2java/expected/complex/dmn/dotproduct/ModelElementRegistry.java

public class ModelElementRegistry extends com.gs.dmn.runtime.discovery.ModelElementRegistry {
    public ModelElementRegistry() {
        // Register elements from model 'DotProduct'
        register("A", "A");
        register("B", "B");
        register("CalculateDotProduct", "CalculateDotProduct");
        register("Componentwise", "Componentwise");
        register("Componentwise", "Componentwise4_iterator");
        register("DotProduct", "DotProduct");
        register("Product", "Product");
    }
}

File: dmn-test-cases/standard/tck/1.1/cl3/0031-static-user-defined-functions/translator/expected/kotlin/dmn/ModelElementRegistry.kt


class ModelElementRegistry : com.gs.dmn.runtime.discovery.ModelElementRegistry {
    constructor() {
        register("Circumference", "Circumference")
        register("'fn invocation complex parameters'", "FnInvocationComplexParameters")
        register("'fn invocation named parameters'", "FnInvocationNamedParameters")
        register("'fn invocation positional parameters'", "FnInvocationPositionalParameters")
        register("'fn library'", "FnLibrary")
        register("inputA", "InputA")
        register("inputB", "InputB")
    }
}